src/providers/slack.ts

```diff
@@ -45,9 +60,24 @@ export class Slack implements OAuth2Provider {
 interface TokenResponseBody {
 	access_token: string;
 	id_token: string;
+	refresh_token?: string;
+	expires_in?: number;
+}
+
+interface RefreshTokenResponseBody {
```

Comment:
Does it not create a new refresh token?

> Use the new refresh_token when you need to refresh the tokens once again. 

https://api.slack.com/authentication/rotation#refresh